apps/tutorial_servo_control/main.cpp

```diff
@@ -0,0 +1,13 @@
+
+AnalogIn potVoltageIn(PA_0);

```

Comment:
you likely need #includes for this file. maybe it compiles bc mbed.h is already included. even if thats the case, still put a #include "mbed.h" at the top